test/models/ETMPPayloadSpec.scala

```diff
@@ -257,7 +257,7 @@ class ETMPPayloadSpec extends AnyWordSpec with Matchers {
       | "latePaymentPenalties": [
       |     {
       |       "type": "additional",
-      |       "reason": "VAT_NOT_PAID_AFTER_30_DAYS",
```

Comment:
Not sure why the reason "VAT_NOT_PAID_AFTER_30_DAYS" was replaced by "OFFICERS_ASSESSMENT_NOT_PAID_WITHIN_15_DAYS"

Reply:
The change wasn't needed. I forgot to revert this back once I had made the changes to the ReasonsEnum.